app/templates/php_script.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:proactive:jobdescriptor:3.9"
+        xsi:schemaLocation="urn:proactive:jobdescriptor:3.9 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.9/schedulerjob.xsd"
+        name="PHP Prototype"
+        priority="normal"
+        onTaskError="continueJobExecution"
+        maxNumberOfExecution="2"
+>
+    <taskFlow>
+        <task name="PHP_TASK">
+            <description>
+                <![CDATA[ Executes a php script. The PHP script be changed through the variable 'pathname' in the task variables. ]]>
+            </description>
+            <variables>
+                <variable name="pathname" value="script.php" inherited="false" />
+                <variable name="phpArguments" value="" inherited="false" />
+            </variables>
+            <genericInformation>
+                <info name="TASK.ICON" value="https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/PHP-logo.svg/100px-PHP-logo.svg.png"/>
```

Comment:
Why commit the image if we get the logo from wikipedia?

Reply:
It was part of the requirements.